packages/workspace/src/tasks-runner/run-command.ts

```diff
@@ -387,10 +388,9 @@ function addTasksForProjectDependencyConfig(
         const depProject = projectGraph.nodes[
           dep.target
         ] as ProjectGraphProjectNode;
-        if (
-          depProject &&
-          projectHasTarget(depProject, dependencyConfig.target)
-        ) {
+        const hasTarget =
+          depProject && projectHasTarget(depProject, dependencyConfig.target);
+        if (depProject && hasTarget) {
```

Comment:
I don't think there's reason to change this line anymore. `hasTarget` is only true if `depProject` is truthy so you're checking `depProject` twice. I'd leave it as is.

Reply:
Agree, simple revert :)